app/views/admin/_merch_form.html.erb

```diff
@@ -0,0 +1,5 @@
+<%= form_with model: [:admin, @merchant], local: true  do |form| %>
+    <%= form.label :name %>
+    <%= form.text_field :name %>
+    <%= form.submit %>
+<% end %>
```

Comment:
So glad we remembered to do a partial here when we realized it was a duplicated form! 👏👏👏